packages/subgraph/tests/admin/admin.test.ts

```diff
@@ -107,7 +107,7 @@ test('Run Admin plugin (handleProposalCreated) mappings with mock event', () =>
   );
 
   // check actions
-  for (let index = 0; index < actions.length; index++) {
+  for (let index = 0; index < [action].length; index++) {
```

Comment:
Why do we loop over an array of length 1?

Reply:
I didnt rewrite the test, I just changed from the local function to the ones in the library, adapting where necessary

Reply:
No worries! For me, this is an example where we can improve code quality by removing unnecessary/redundant code easily.

Reply:
Yep the subgraph in particular needs a lot of love when we split it

Reply:
would we ever be testing multiple actions in this test?

Reply:
@jordaniza id does not bring too much value, I updated the test a bit tho